### ISSUE TITLE
Let empty bulk refresh call fail silently

### ISSUE THREAD

**Issue by admturner** [washingtonstateuniversity/WSUWP-Plugin-A11y-Status#16]
If a user selected the bulk user "Refresh a11y status" action without actually selecting any users the plugin throws the following errors, because there is no data for the message handler: 

~~~
Notice: Undefined index: success in /srv/www/wordpress-beta/public_html/wp-content/plugins/wsuwp-a11y-status/includes/class-setup.php on line 673

Notice: Undefined index: fail in /srv/www/wordpress-beta/public_html/wp-content/plugins/wsuwp-a11y-status/includes/class-setup.php on line 684
~~~

Fix this by either exiting the message handler if there's nothing to display, or exiting the batch handler early if no users are selected.
